assets/app/scripts/services/imageStreamResolver.js

```diff
@@ -27,8 +31,10 @@ angular.module('openshiftConsole')
         }
 
         var imageStreamImagePromise = DataService.get("imagestreamimages", imageStreamImageRef, context);
-        imageStreamImagePromise.then(function(image) {
-          imagesByDockerReference[dockerRef] = image;
+        imageStreamImagePromise.then(function(imageStreamImage) {
```

Comment:
Maybe it always resolves. My fear was it would result in a null pointer now where it wouldn't before.